src/aiida_wannier90_workflows/workflows/projwfcbands.py

```diff
@@ -119,10 +118,6 @@ def get_builder_from_protocol(  # pylint: disable=arguments-differ
         # Prepare workchain builder
         builder = cls.get_builder()
 
-        protocol_inputs = cls.get_protocol_inputs(
-            protocol=protocol, overrides=overrides
-        )
-
```

Comment:
Hi @t-reents , thanks for the PR.

These lines should remain. They are used to read the protocol parameters from the file, `projwfcbands.yaml`. For the moment, the file only specifies `clean_workdir`. I think that's why you get identical results even after you remove them. However, more parameters may be added in the future.

Reply:
but dont you get that from "projwfc_builder = ProjwfcBaseWorkChain.get_builder_from_protocol(
            projwfc_code, protocol=protocol, overrides=projwfc_overrides
        )" ?

Reply:
Hi @superstar54 

I see the point that one might create a specific protocol for this WorkChain. I also mentioned this in the PR description. My point was that this WorkChain doesn't implement any additional inputs and basically inherits the `PwBandsWorkChain` only. I assumed that one would simply provide the individual changes via the overrides. 

But yeah, it makes sense to have the possibility to specify a protocol to overwrite the defaults of the parents. Otherwise, one would always need to provide adjusted protocols for `PwBandsWorkChain` and `ProjwfcBaseWorkChain`.

Reply:
@superstar54 Coming back to the point, that the `ProjwfcBandsWorkChain` doesn't have additional inputs, I'd suggest to load the protocol inputs and merge the relevant parts with the overrides for the subsequent calls of `PwBandsWorkChain` and `ProjwfcBaseWorkChain`. In this way, we still have the possibility to provide custom protocols but also fix the initial problem that the transformed values get overwritten by the Python base types.

Reply:
Fine to me. If you do this, please also remove the `projwfcbands.yaml` file, and then add a detailed comment in the code that the WorkChain uses the parent protocol setting.

It would be good that @qiaojunfeng could have a quick look at it.